packages/carbon-web-components/src/components/slug/slug-data-table-story.ts

```diff
@@ -0,0 +1,600 @@
+          <cds-table-cell>HTTP</cds-table-cell>
+          <cds-table-cell>443</cds-table-cell>
+          <cds-table-cell>Round robin</cds-table-cell>
+          <cds-table-cell>Maureen's VM Groups</cds-table-cell>
+          <cds-table-cell><cds-link>Starting</cds-link></cds-table-cell>
+        </cds-table-row>
+        <cds-table-expanded-row>
+          <h6>Expandable row content</h6>
+          <div>Description here</div>
+        </cds-table-expanded-row>
+        <cds-table-row>
+          <cds-table-cell>Load Balancer 2</cds-table-cell>
+          <cds-table-cell>HTTP</cds-table-cell>
+          <cds-table-cell>80</cds-table-cell>
+          <cds-table-cell>DNS delegation</cds-table-cell>
+          <cds-table-cell>Andrew's VM Groups</cds-table-cell>
+          <cds-table-cell><cds-link>Active</cds-link></cds-table-cell>
+        </cds-table-row>
+        <cds-table-expanded-row>
+          <h6>Expandable row content</h6>
+          <div>Description here</div>
+        </cds-table-expanded-row>
+        <cds-table-row>
+          <cds-slug alignment="bottom-left">${content}${actions}</cds-slug>
+          <cds-table-cell>Load Balancer 6</cds-table-cell>
+          <cds-table-cell>HTTP</cds-table-cell>
+          <cds-table-cell>3000</cds-table-cell>
+          <cds-table-cell>Round robin</cds-table-cell>
+          <cds-table-cell>Marc's VM Groups</cds-table-cell>
+          <cds-table-cell
+            ><cds-link disabled>Disabled</cds-link></cds-table-cell
+          >
+        </cds-table-row>
+        <cds-table-expanded-row>
+          <h6>Expandable row content</h6>
+          <div>Description here</div>
+        </cds-table-expanded-row>
+        <cds-table-row>
+          <cds-slug alignment="bottom-left">${content}${actions}</cds-slug>
+          <cds-table-cell>Load Balancer 4</cds-table-cell>
+          <cds-table-cell>HTTP</cds-table-cell>
+          <cds-table-cell>443</cds-table-cell>
+          <cds-table-cell>Round robin</cds-table-cell>
+          <cds-table-cell>Mel's VM Groups</cds-table-cell>
+          <cds-table-cell><cds-link>Starting</cds-link></cds-table-cell>
+        </cds-table-row>
+        <cds-table-expanded-row>
+          <h6>Expandable row content</h6>
+          <div>Description here</div>
+        </cds-table-expanded-row>
+        <cds-table-row>
+          <cds-table-cell>Load Balancer 5</cds-table-cell>
+          <cds-table-cell>HTTP</cds-table-cell>
+          <cds-table-cell>80</cds-table-cell>
+          <cds-table-cell>DNS delegation</cds-table-cell>
+          <cds-table-cell>Ronja's VM Groups</cds-table-cell>
+          <cds-table-cell><cds-link>Active</cds-link></cds-table-cell>
+        </cds-table-row>
+        <cds-table-expanded-row>
+          <h6>Expandable row content</h6>
+          <div>Description here</div>
+        </cds-table-expanded-row>
+      </cds-table-body>
+    </cds-table>
+  `;
+};
+
+export const _SlugWithRadio = () => {
```

Comment:
```suggestion
export const _SlugWithRadioSelection = () => {
```

just to match with the React storybook